src/muya/lib/ui/imageToolbar/index.js

```diff
@@ -135,6 +154,12 @@ class ImageToolbar extends BaseFloat {
         this.muya.contentState.updateImage(this.imageInfo, 'data-align', item.type)
         return this.hide()
       }
+      case 'open': {
+        if (imageIsLocal) {
+          this.muya.contentState.openImage(this.imageInfo, this.realFilePath)
```

Comment:
The path is (always) prefixed wih `file://` and we have to remove this to get a valid absolute file path.